examples/pytorch/speech-recognition/run_speech_recognition_ctc.py

```diff
@@ -207,15 +207,31 @@ class DataTrainingArguments:
     )
     max_duration_in_seconds: float = field(
         default=20.0,
+        metadata={
+            "help": "Filter audio files in the training set that are longer than `max_duration_in_seconds` seconds"
+        },
+    )
+    min_duration_in_seconds: float = field(
+        default=0.0,
+        metadata={
+            "help": "Filter audio files in the training set that are shorter than `min_duration_in_seconds` seconds"
+        },
+    )
+    max_eval_duration_in_seconds: float = field(
+        default=None,
         metadata={
             "help": (
-                "Filter audio files that are longer than `max_duration_in_seconds` seconds to"
-                " 'max_duration_in_seconds`"
+                "Filter audio files in the evaluation set that are longer than `max_eval_duration_in_seconds` seconds"
             )
         },
     )
-    min_duration_in_seconds: float = field(
-        default=0.0, metadata={"help": "Filter audio files that are shorter than `min_duration_in_seconds` seconds"}
+    min_eval_duration_in_seconds: float = field(
```

Comment:
```suggestion
    min_eval_duration_in_seconds: Optional[Union[float, bool]] = field(
```
For backwards compatibility I'd leave it exactly like is, but also allow one to set it simply to `False` - see comment below to better understand